ports/libexif/portfile.cmake

```
@@ -1,24 +1,36 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO libexif/libexif
-    REF v0.6.24
+    REF "v${VERSION}"
     SHA512 eac1b5220ca0e02370837a0d78a6d38e91c5afa0956d4196b26a8d2a8a2c5dea18d58c0e473285f278653c3863923241651b7dff4d007cc46385eb29ea188330
     HEAD_REF master
     PATCHES
-        add-missing-_stdint-h.patch
+        fix-ssize.patch
 )
 
-file(COPY "${CMAKE_CURRENT_LIST_DIR}/CMakeLists.txt" DESTINATION "${SOURCE_PATH}")
-file(COPY "${CMAKE_CURRENT_LIST_DIR}/config.h.cmake" DESTINATION "${SOURCE_PATH}")
-file(COPY "${CMAKE_CURRENT_LIST_DIR}/libexif.def"    DESTINATION "${SOURCE_PATH}")
+vcpkg_list(SET options)
+if("nls" IN_LIST FEATURES)
+    vcpkg_list(APPEND options "--enable-nls")
+else()
+    set(ENV{AUTOPOINT} true) # true, the program
+    vcpkg_list(APPEND options "--disable-nls")
+endif()
```

```suggestion
set(ENV{AUTOPOINT} true) # true, the program
vcpkg_list(SET options)
if("nls" IN_LIST FEATURES)
    vcpkg_list(APPEND options "--enable-nls")
else()
    vcpkg_list(APPEND options "--disable-nls")
endif()
```
We shall not run autopoint because we don't run `gnulib-tool`.
https://www.gnu.org/software/gnulib/manual/html_node/gettextize-and-autopoint.html

---

Hm, did it build before this change? (autoconf step).
Then sorry for the confusion - this will stay on my watch list.

---

The code logic seems to be the optimal solution but it build before `vcpkg_configure_make`.